bitnami/odoo/templates/deployment.yaml

```diff
@@ -24,8 +31,14 @@ spec:
       {{- end }}
     spec:
       {{- include "odoo.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+      {{- end }}
+      serviceAccountName: {{ .Values.serviceAccountName }}
```

Comment:
We're using `serviceAccount.name` and `serviceAccount.create` parameter in other charts which allow the user to decide whether to create/user a custom service account to run the application or not. I think we should follow the same approach here.

Reply:
Ok, applied